src/settings/MatchProfiles/MatchProfiles.js

```diff
@@ -133,7 +146,7 @@ export class MatchProfiles extends Component {
     } = this.props;
 
     const urlQuery = queryString.parse(search);
-    const searchTerm = (urlQuery.query || '').trim();
+    const searchTerm = trimSearchTerm((urlQuery.query || '').trim());

```

Comment:
Consider moving `.trim()` call in `trimSearchTerm` function as it looks weird to trim 2 times

Reply:
I would insist to move to `trimSearchTerm`.